walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,6 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
-    return "Hello " + name;
+    return "Hello " + name.trim();
```

Comment:
Nice! You don't have to change anything but consider what would happen if the person's name had a space in it. Would this still work?

Reply:
Oh, that is not my doing. It was part of the instructions from the tutorial.